README.md

```diff
@@ -0,0 +1,28 @@
+# lightning-address-details-proxy
+
+This proxy exists to simplify requests to lightning address providers.
+
+- Many ln addresses don't support CORS, which means fetching the data directly in a browser environment will not always work.
+- Two requests are required to retrieve lnurlp and keysend data for a lightning address. The proxy will do these for you with a single request.
+
+# API
+
+## Getting keysend and LNURLp info
+
+`GET /lightning-address-details?ln=<lightning_address>`
+
+### Example
+
+GET https://lnaddressproxy.getalby.com/lightning-address-details?ln=hello@getalby.com
+
+## Requesting a LNURLp invoice
+
+`POST /generate-invoice?ln=<lightning_address>&amount=<amount_in_millisats>&comment=<http_encoded_comment>`
```

Comment:
```suggestion
`GET /generate-invoice?ln=<lightning_address>&amount=<amount_in_millisats>&comment=<http_encoded_comment>`
```
LNURL uses GET thus we use also get

in the docs we should reference also the LNURL params

Reply:
@bumi  thanks, added